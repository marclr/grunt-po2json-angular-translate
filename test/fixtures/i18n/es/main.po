msgid ""
msgstr ""

msgid "HELLO_WORLD"
msgstr "Hola mundo"

msgid "INTRO"
msgstr "Introducción"

msgid "DESCRIPTION"
msgstr "Descripción"
