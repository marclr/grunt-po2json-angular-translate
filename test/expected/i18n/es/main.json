{
   "HELLO_WORLD": "Hola mundo",
   "INTRO": "Introducción",
   "DESCRIPTION": "Descripción"
}